src/ipips/ipip-0402.md

```diff
@@ -235,9 +230,14 @@ single request.
 
 ## Test fixtures
```

Comment:
:information_source:  A detailed list of compliance checks for `dag-scope` and `entity-bytes` can be found in  [`gateway-conformance v0.2.0 → trustless_gateway_car_test.go`](https://github.com/ipfs/gateway-conformance/blob/v0.2.0/tests/trustless_gateway_car_test.go). I've made another pass over them, and pushed https://github.com/ipfs/specs/pull/402/commits/4ebbb9642439abf2b97ebc91e53c79f9d979c7b4  which adds CIDs, CARs, and short summary of each fixture to this IPIP.